src/service/xhr-impl.js

```diff
@@ -407,13 +427,11 @@ function isRetriable(status) {
  */
 export function assertSuccess(response) {
   return new Promise((resolve, reject) => {
-    if (response.status < 200 || response.status >= 300) {
-      /** @const {!Error} */
-      const err = user().createError(`HTTP error ${response.status}`);
-      err.response = response;
-      if (isRetriable(response.status)) {
-        err.retriable = true;
-      }
+    const status = response.status;
+    if (status < 200 || status >= 300) {
+      const retriable = isRetriable(status);
+      const err = user().createCustomError(
```

Comment:
The only thing sent in the error report is the message first argument. The other information is just tacked on to the error object and not sent, so the full response won't be logged.
Also, `createError` doesn't report errors so this error isn't logged anyway.